### ISSUE TITLE
[BIM] Project Setup Manager

### ISSUE THREAD

**Issue by hoshengwei** [yorikvanhavre/BIM_Workbench#135]
when I try to use the BIM  Project Setup Tool to establish the BIM project
the original design of the code is to put `Outline` `Human Figure` `Axes` etc. in the group folder which default name is `Building Layout`

but if I leave the `Gross building Length / width`  blank (0)
it will not create that group folder to put those obj.
in `BimProjectManager.py` line 166
```python
grp = FreeCAD.ActiveDocument.addObject("App::DocumentObjectGroup")
grp.Label = translate("BIM", "Building Layout")
building.addObject(grp)
```
maybe need to move those code above outside the if condition?

**Comment by hoshengwei** [comment]
I open a PR #136  to fix the  problem  above

**Comment by yorikvanhavre** [comment]
Fixed with #136 (FYI: in your commit message, you can write something like "fixes #136" and it automatically closes the issue when merged)
